docs/release-notes.rst

```diff
@@ -6,6 +6,28 @@
  Release Notes
 ###############
 
+**************
+ Version 0.27
+**************
+
+Version 0.27.0
+==============
+
+**Release Date:** January 09, 2024
+
+**Breaking Changes**
+
+-  Experiments: Allow empty model definitions when creating experiments.
```

Comment:
This seems like maybe it shouldn't count as breaking. @corban-beaird?

Reply:
Yea, that's a fair point, the second bullet point is the breaking change, the first is the reason why the break is occuring

Reply:
@corban-beaird should the first bullet be removed all together or moved to the `improvements` section?